runtime/src/iree-amd-aie/aie_runtime/Utils/ChannelBdIdGenerator.h

```diff
@@ -41,11 +50,25 @@ class ChannelBdIdGenerator {
   /// reused.
   void releaseBdId(uint32_t bdId) { assignedBdIds.erase(bdId); }
 
+  uint32_t getAvailableBdIdNum(uint32_t channel) {
```

Comment:
```suggestion
  uint32_t getNumAvailableBdIds(uint32_t channel) {
```
This is more consistent with naming in the codebase (`type.getNumElements`, `getNumArguments`, etc)